fastqpreprocessing/src/htslib_tagsort.cpp

```diff
@@ -462,3 +464,9 @@ std::vector<std::string> create_sorted_file_splits_htslib(INPUT_OPTIONS_TAGSORT
   return alignment_reader.partial_filenames();
 }  // function
 
+void crash(std::string msg)
```

Comment:
In cleaning up the build process, I realized TagSort can be completely detached from the fastq programs, with crash() from utilities.cpp being the sole exception. With crash() duplicated into here, TagSort could be moved into its own separate directory.